spec/lang/step/operators.md

```diff
@@ -48,9 +48,15 @@ impl<M: Memory> Machine<M> {
                 ret((Value::Int(result), Type::Int(int_ty)))
             }
             Transmute(new_ty) => {
+                // Question: Why is this not a well-formedness constraint?
+                // `core::intrinsics::transmute` states:
+                // "Both types must have the same size. Compilation will fail if this is not guaranteed."
                 if old_ty.size::<M::T>() != new_ty.size::<M::T>() {
                     throw_ub!("transmute between types of different size")
-                };
```

Comment:
And here I am not entirely sure why this is UB and not a well formedness violation

Reply:
Ah wait, I found the comment in MiniRust specifying that this is meant to be like that